sound/soc/sof/topology.c

```diff
@@ -506,32 +506,32 @@ static int get_token_uuid(void *elem, void *object, u32 offset, u32 size)
 	return 0;
```

Comment:
that's not super-elegant. In the case of scalar tokens we return a value but in the case of  larger items we return 0, but this zero is meaningless.

Reply:
yes, I fixed this to rename get_token to load_token and added a new get_token